test/PaletteInsightAgent.UnitTests/ThreadInfo/ThreadInfoAgentTest.cs

```diff
@@ -1,92 +1,53 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-
+using PaletteInsightAgent.ThreadInfoPoller;
 namespace PaletteInsightAgentTests.ThreadInfo
 {
     /// <summary>
     /// Summary description for ThreadInfoAgentTest
     /// </summary>
     [TestClass]
-    public class ThreadInfoAgentTest
+    public class ThreadInfoAgentTest 
     {
-        public ThreadInfoAgentTest()
+        public void ThreadInfoTest()
```

Comment:
This is no longer a constructor... this line should look like this:
```suggestion
        public ThreadInfoAgentTest()
```